contentmanagement/proposal.md

```diff
@@ -0,0 +1,59 @@
+# Proposal for Incubation
+
+## Subproject name
+
+Jupyter Content Management Extensions (jupyter-incubator/contentmanagement)
+
+## Development team and Advocate
+
+Subproject development team:
+
+* Justin Tyberg, IBM (`@jtyberg `)
+* Dan Gisolfi, IBM (`@vinomaster`)
+* Peter Parente, IBM (`@parente`)
+
+Steering Council Advocate:
+
+* Brian Granger (`@ellisonbg`)
+
+## Subproject goals, scope and functionality
+
+### Concept 
+The Jupyter Notebook allows for the publishing, editing, organizing and deleting of content from a central web interface. Depending on an individual's workflow and collaboration requirements when working on notebook related projects, the collection of content management features they desire may differ from other users.
+
+This incubation proposal pertains to the creation of a package of self-installable content management features that extend the baseline Jupyter Notebook. 
+
+### Goals
+* Establish a collection of content management extensions that can be installed to customize a user instance of a Jupyter Notebook.
+* Allow 1..n content management extensions to be combined to customize an instance of a Jupyter Notebook.  
+
+### Scope
+
+Refactor content management features that were used to build the [IBM Knowledge Anyhow Workbench](https://knowledgeanyhow.org) technology preview, and potentially graduate them to a Jupyter subproject, or integrate them into the [Jupyter Interactive Notebook](jupyter/notebook) repo.
+
+### Functionality
+
+As outlined in this [blog post](http://blog.ibmjstart.net/2015/08/20/jupyter-notebooks-content-management-contributions/), the intent of this proposal is to provide support for:
+
+* Including secondary notebooks by reference. 
+* Table of contents and intra-notebook navigation.
+* Adding a search button on any or all of the user views (notebook, file-tree, text editor).
+* Downloading file bundles (.zips) from the Jupyter dashboard
+* Importing any web-accessible notebook via an omnibox
```

Comment:
I'm slightly afraid on this one that user will not understand that they are actually executing code from the internet and that this could harm their computer. I just want to be careful about these point during implementation about what "import" is, but have no thing against this point.